docs/yul.rst

```diff
@@ -336,248 +682,207 @@ We will use a destructuring notation for the AST nodes.
-+---------------------------------------------+-----------------------------------------------------------------+
-| combineu64tou256(x1:u64, x2:u64, x3:u64,    | combine four u64's into a single u256                           |
-| x4:u64) ‑> (x:u256)                         |                                                                 |
-+---------------------------------------------+-----------------------------------------------------------------+
-| keccak256(p:u256, s:u256) ‑> v:u256         | keccak(mem[p...(p+s)))                                          |
-+---------------------------------------------+-----------------------------------------------------------------+
-| *Object access*                             |                                                                 |
-+---------------------------------------------+-----------------------------------------------------------------+
-| datasize(name:string) ‑> size:u256          | size of the data object in bytes, name has to be string literal |
-+---------------------------------------------+-----------------------------------------------------------------+
-| dataoffset(name:string) ‑> offset:u256      | offset of the data object inside the data area in bytes,        |
-|                                             | name has to be string literal                                   |
-+---------------------------------------------+-----------------------------------------------------------------+
-| datacopy(dst:u256, src:u256, len:u256)      | copy len bytes from the data area starting at offset src bytes  |
-|                                             | to memory at position dst                                       |
-+---------------------------------------------+-----------------------------------------------------------------+
-
-Backends
---------
-
-Backends or targets are the translators from Yul to a specific bytecode. Each of the backends can expose functions
-prefixed with the name of the backend. We reserve ``evm_`` and ``ewasm_`` prefixes for the two proposed backends.
-
-Backend: EVM
-------------
-
-The EVM target will have all the underlying EVM opcodes exposed with the `evm_` prefix.
-
-Backend: "EVM 1.5"
-------------------
-
-TBD
-
-Backend: eWASM
---------------
-
-TBD
+.. _opcodes:
+
+EVM Dialect
+-----------
+
+The default dialect of Yul currently is the EVM dialect parameterized
+with a version of the EVM. The only type available in this dialect
+is ``u256``, the 256-bit native type of the Ethereum Virtual Machine.
+Since it is the default type of this dialect, it can be omitted.
+
+The following table lists all builtin functions
+(depending on the EVM version) and provides a short description of the
+semantics of the function / opcode.
+This document does not want to be a full description of the Ethereum virtual machine,
+so please refer to a different document if you are interested in the precise semantics.
```

Comment:
I would suggest to link the yellow paper, but it's still not up to date, is it? *Is* there actually a good choice for  such a "different document" right now :-D? If so we should in fact link it...
We could also remove this sub-clause...

Reply:
I do not want to impose the burden of the yellow paper to people.